chargebee/src/main/java/com/chargebee/android/billingservice/BillingClientManager.kt

```diff
@@ -157,16 +156,12 @@ class BillingClientManager constructor(
     /* Purchase the product: Initiates the billing flow for an In-app-purchase  */
     fun purchase(
         product: CBProduct,
-        customerID: String = "",
         purchaseCallBack: CBCallback.PurchaseCallback<String>
     ) {
         this.purchaseCallBack = purchaseCallBack
         this.product = product
         val skuDetails = product.skuDetails
 
-        if (!(TextUtils.isEmpty(customerID))) {
-            this.customerID = customerID
-        }
```

Comment:
Do we not need the customerID anymore?

Reply:
We are having this customerId checks which to work on backward compatible versions. will remove it once we remove the old `purchaseProduct()` method. what do you think?